pyproject.toml

```diff
@@ -8,7 +8,7 @@ version = "0.9.0"
 description = "An eco-routing tool build upon RouteE-Powertrain"
 readme = "README.md"
 documentation = "nrel.github.io/routee-compass"
-requires-python = ">=3.9"
+requires-python = ">=3.11"
```

Comment:
Sorry I should've been more clear but we can still support python 3.9 and 3.10 as build targets for our package but it's just that we have to [install python 3.11 in our CI ](https://github.com/NREL/routee-compass/blob/beb3f78d9854be3e4f909fe8c663967fc82b0de5/.github/workflows/python-release.yaml#L36).

Reply:
This is the specific warning:

```
Warning: cibuildwheel 3 will require Python 3.11+, please upgrade the Python version used to run cibuildwheel. This does not affect the versions you can target when building wheels. See: https://cibuildwheel.pypa.io/en/stable/#what-does-it-do
```